### DISCUSSION TITLE
Enhancement of PreferencesDialog

### DISCUSSION THREAD

**Discussion by tozGitHub** [WinEdt-Team/WinEdt-Support#37] [1.3 Ideas and features requests]
Dear Alex, dear community,

I was thinking that
```
PreferencesDialog;
```
should behave like 
```
DocumentSettings( Page );
```
That is, it should have an argument showing the page. The reason for this is that it might help automate to generate the images for the help file.

Best,
Thomas

**Comment by Alex-Simonic** [comment]
For what it is worth you now have:

`PreferencesDialog( Page: 0..11 );`

and

`DocumentSettings( Page: 0..6 );`

with 0 interpreted as default (do not reset the page index)...
